pom.xml

```diff
@@ -981,8 +954,6 @@
                 <unused>org.apache.logging.log4j:log4j-1.2-api:jar:*</unused>
                 <unused>org.apache.logging.log4j:log4j-slf4j2-impl:jar:*</unused>
                 <unused>org.apache.logging.log4j:log4j-web:jar:*</unused>
-                <!-- This should be removed upon completion of migrating junit 4 to 5 -->
-                <unused>org.junit.vintage:junit-vintage-engine:jar:*</unused>
```

Comment:
I think there is also an entry in the `ignoredUsedUndeclaredDependencies` section that can be removed like this one.

Reply:
Additionally in the pom, in the `maven-checkstyle-plugin` section, we might be able to remove some rules that refference junit4 imports or add one/some that ensure only junit jupiter imports are used.

Reply:
So removing powermock mean we can remove usage of all old junit apis?  If so that is nice, it would be nice if the old stuff was no longer there in autocomplete in IDE.